code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -223,13 +223,13 @@
 /mob/living/carbon/human/proc/generate_death_examine_text()
 	var/mob/dead/observer/ghost = get_ghost(TRUE, TRUE)
 	var/t_He = p_they(TRUE)
-	var/t_his = p_their()
+	// var/t_his = p_their() - SKYRAT EDIT - UNUSED VAR
 	var/t_is = p_are()
 	//This checks to see if the body is revivable
 	if(key || !getorgan(/obj/item/organ/brain) || ghost?.can_reenter_corpse)
 		return "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life...</span>"
 	else
-		return "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life and [t_his] soul has departed...</span>"
+		return "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life and rigor mortis is starting to set in. Revival is unlikely...</span>" // SKYRAT EDIT - ORIGINAL  ...signs of life and [t_his] soul has departed...
```

Comment:
Saying 'unlikely' is going to lead to confusion. Either say revival is no longer possible, or cut out the bit related to revival entirely

Reply:
I said unlikely because I thought "his soul has departed" also applied to ghosts who had disconnected, but could reconnect. This isn't the case, will change